### PR TITLE
Fix databricks using original py filename

### PR DESCRIPTION
**Pull request by basvdl**
## Summary:

In the previous PR a feature was added to upload an artefact using the original filename. This impacts the deploy to databricks step. This PR implements the same flag.

## Breaking Change:

None

## Checklist:
  - [x] The code change is tested and works locally.
  - [x] There is no commented out code in this PR.
  - [x] There are no new TODOs in this PR

If exposed functionality or configuration variables are added/changed:
  - [x] Documentation added/updated

### CODE REVIEW

File: docs/_deployment-steps/deploy_to_databricks.md [hunk 1]
```diff
@@ -42,6 +42,7 @@ This should be after the [upload_to_blob](upload-to-blob) task if used together
 | `jobs[].run_stream_job_immediately` (optional) | Whether or not to run a stream job immediately | `True` or `False`. Defaults to `True`.
 | `jobs[].is_batch` (optional) | Designate job as an unscheduled batch | `True` or `False`. Defaults to `False`.
 | `jobs[].arguments` (optional) | Key value pairs to be passed into your project | defaults to no arguments
+| `jobs[].use_original_python_filename` [optional] | If you uploaded multiple unique Python files using the `use_original_python_filename` in the `publish_artifact` job, use this flag here too. Only impacts Python files.
```
Comment (danielvdende) [inline docs/_deployment-steps/deploy_to_databricks.md:45]: nit: could you make the brackets around `optional` parentheses? :D that's what the other optional ones have too.

Comment (danielvdende) [inline docs/_deployment-steps/deploy_to_databricks.md:45]: On a general note: would it be a good idea to add an example here? For me, the explanation is good, but I still don't quite understand the impact on the naming of my files. I think adding an example somewhere in the docs would help.

**Review by danielvdende — CHANGES_REQUESTED** [review]
2 minor comments on the docs :)

File: docs/_deployment-steps/deploy_to_databricks.md (hunk 1 above)
Comment (basvdl) [reply docs/_deployment-steps/deploy_to_databricks.md]: This is a bit random throughout the project. But for this file I changed it to parentheses.

Comment (basvdl) [reply docs/_deployment-steps/deploy_to_databricks.md]: I have added a table that explains the behaviour of the flag. The table is also added to the publish-artifact docs

**Review by danielvdende — APPROVED** [review]

**Pull request merged**